src/app/src/main/kotlin/org/vaccineimpact/orderlyweb/app_start/routing/web/WebUserRouteConfig.kt

```diff
@@ -14,6 +14,12 @@ object WebUserRouteConfig : RouteConfig
                     contentType = ContentTypes.json)
                 .json()
                 .transform()
-                .secure(usersManage)
+                .secure(usersManage),
+            WebEndpoint("/users/",
```

Comment:
Should this be `/users/` or something more specific, like `/emails/`? It's just for returning valid user emails for the autocomplete...

Reply:
If it's just returning emails I'd probably expect a more specific route, since we already have endpoint which are returning user models, which in a RESTful way I'd expect to be the resource I'd find at /users/. But since this *isn't* a part of the REST API and is just for the web app's internal use I guess it isn't so important!

Reply:
Actually, having said that, maybe we don't have anything returning the User model just now, as there's now a ReportReaderViewModel instead. Although maybe that will change once we've done the Admin screens.
I still think /users/emails might be a bit clearer!